Automattic/woocommerce-payments
# refactor: settings reducer redundant action types

Fixes #

#### Changes proposed in this Pull Request









I think it would be nice to clean up a bit the action types in the settings reducer.
This would allow for a smaller bundle size, since the reducer wouldn't have to have different `case`s (less branching logic) and we wouldn't need to have multiple action types for each setting.

#### Testing instructions





- Ensure you have the latest version of the dev tools: `cd docker/wordpress/wp-content/plugins/woocommerce-payments-dev-tools && git checkout trunk && git pull`
- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Ensure you have the "Add UPE additional payment methods" flag enabled in WCPay Dev Tools
- Go to your WCPay Home: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Settings are all still saving, even after a page refresh

-------------------
